include/extractor/extraction_containers.hpp

```diff
@@ -49,9 +47,15 @@ class ExtractionContainers
     NameCharData name_char_data;
     NameOffsets name_offsets;
     // an adjacency array containing all turn lane masks
-    RestrictionsVector restrictions_list;
     WayIDStartEndVector way_start_end_id_list;
+
     unsigned max_internal_node_id;
+
+    // list of restrictions before we transform them into the output types
+    std::vector<InputConditionalTurnRestriction> restrictions_list;
```

Comment:
Input restrictions are generated during parsing and store OSM IDs. Only after we performed our re-mapping, we can assign the actual restrictions to be used later on.